core/.vscode/settings.json

```diff
@@ -0,0 +1,47 @@
+{
+  "python.pythonPath": "${workspaceFolder}/.venv",
+  // PEP8 settings (https://www.python.org/dev/peps/pep-0008/)
+  "editor.tabSize": 4,
+  "editor.insertSpaces": true,
+  "editor.wordWrapColumn": 88,
+  "editor.wordWrap": "wordWrapColumn",
+  // Single newline at end of file
+  "files.trimTrailingWhitespace": true,
+  "files.insertFinalNewline": true,
+  "files.trimFinalNewlines": true,
+  // Always format with black on save
+  "editor.formatOnSave": true,
+  "python.formatting.provider": "black",
+  // Linting and code quality
+  "python.linting.enabled": true,
+  "python.linting.lintOnSave": true,
+  "python.linting.pylintEnabled": false,
+  "python.linting.mypyEnabled": true,
+  "python.linting.flake8Enabled": false,
+  "python.linting.pylamaEnabled": true,
+  "python.linting.flake8Args": [
+    "--verbose"
+  ],
+  // Unit testing (only one framework can be active)
+  "python.testing.unittestEnabled": false,
```

Comment:
This one should be `true` to be ready when we will add some tests.

Reply:
If I leave it at `true`, since there is no test for now we get `No tests discovered, please check the configuration settings for the tests.`. Are we ok with this?

Reply:
Leaving as is for now